api/v1beta1/openstackdataplanenodeset_types.go

```diff
@@ -136,10 +146,20 @@ func (instance *OpenStackDataPlaneNodeSet) InitConditions() {
 
 // GetAnsibleEESpec - get the fields that will be passed to AEE
 func (instance OpenStackDataPlaneNodeSet) GetAnsibleEESpec() AnsibleEESpec {
+	// Build env to include DEPLOY_CONFIG_HASH at the front of the slice
+	var aeeEnv []corev1.EnvVar
+	aeeEnv = append(aeeEnv, corev1.EnvVar{
+		Name: "DEPLOY_CONFIG_HASH",
+		Value: instance.Status.ConfigHash,
```

Comment:
It would probably be better to use a label rather than inserting into the `EnvVars` like this. Happy to discuss further.